crates/re_types_builder/src/objects.rs

```diff
@@ -161,19 +167,54 @@ pub struct Docs {
     ///
     /// See also [`Docs::doc`].
     pub tagged_docs: HashMap<String, Vec<String>>,
+
+    /// Contents of all the files included using `include:<path>`.
+    pub included_files: HashMap<PathBuf, String>,
 }
 
 impl Docs {
     fn from_raw_docs(
+        filepath: &Path,
         docs: Option<flatbuffers::Vector<'_, flatbuffers::ForwardsUOffset<&'_ str>>>,
     ) -> Self {
+        let mut included_files = HashMap::default();
+
+        let include_file = |included_files: &mut HashMap<_, _>, raw_path: &str| {
+            let path: PathBuf = raw_path
+                .parse()
+                .with_context(|| format!("couldn't parse included path: {raw_path:?}"))
+                .unwrap();
+
+            let path = filepath.join(path);
+
+            included_files
+                .entry(path.clone())
+                .or_insert_with(|| {
+                    std::fs::read_to_string(&path)
+                        .with_context(|| {
+                            format!("couldn't parse read file at included path: {path:?}")
+                        })
+                        .unwrap()
+                })
+                .clone()
+        };
+
         // language-agnostic docs
         let doc = docs
             .into_iter()
             .flat_map(|doc| doc.into_iter())
             // NOTE: discard tagged lines!
             .filter(|line| !line.trim().starts_with('\\'))
-            .map(ToOwned::to_owned)
+            .flat_map(|line| {
+                if let Some((_, path)) = line.split_once("include:") {
```

Comment:
I think we should consider making special directives like `include:` stick out a bit more. Consider this comment:

```
/// The magic string can fail in some situations which
/// include: unfortunate positioning of line breaks.
```

You use `\rs` and `\py` already, so maybe `\include:` ?

Backslashes are a bit annoying though because of their use as escapes, so maybe `$rs`, `$include:` etc?

Reply:
Hmmm... I'll make it `\include:` for now and then we can make it all better later on